### PR TITLE
Move to edition "2018"

### PR DESCRIPTION
**Pull request by Hofer-Julian**
For further details see:
https://doc.rust-lang.org/edition-guide/rust-2018/index.html

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -156,5 +155,4 @@ name = "printing"
 
 [[bin]]
 name = "gio_async_tls"
-required-features = ["async-tls"]
-edition = "2018"
+required-features = ["async-tls"]
```
Comment (EPashkin) [inline Cargo.toml:158]: "No newline at end of file"

Comment (Hofer-Julian) [reply Cargo.toml]: Fixed

**Comment by Hofer-Julian** [comment]
The tests are now also green. I think it was an outage of homebrew.

**Comment by sdroege** [comment]
Moved to https://github.com/gtk-rs/gtk-rs/pull/32

**Pull request closed without merging**